### PR TITLE
Add guide for Google Analytics

### PR DESCRIPTION
**Pull request by marcusolsson**
This PR creates a new guide for how to set up Google Analytics for an Obsidian Publish site.

#268

### CODE REVIEW

**Comment by lishid** [comment]
I think we now also support the new google analytics code `G-` in addition to the old `UA-`. Users just have to use the G- code instead.

File: en/Obsidian Publish/Set up Google Analytics.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+[Google Analytics](https://analytics.google.com) is a service that allows you to track traffic on your website. This page explains how to enable Google Analytics for your [[Introduction to Obsidian Publish|Obsidian Publish]] site.
+
+> **Important:** Before you enable Google Analytics, make sure that your local laws and regulations allow you to track your visitors.
+
+## Prerequisites
+
+To use Google Analytics for Obsidian, you need:
+
+- A Universal Analytics (UA) property (not the new Google Analytics 4 property).
```
Comment (marcusolsson) [inline en/Obsidian Publish/Set up Google Analytics.md:9]: @lishid Since both are supported, I think we can remove this preqrequisite then?

Comment (lishid) [reply en/Obsidian Publish/Set up Google Analytics.md:9]: That's correct!

**Review by lishid — APPROVED** [review]

**Comment by lishid** [comment]
Simple and concise as always!

**Pull request merged**